src/index.html

```diff
@@ -20,8 +24,364 @@
       href="styles/index.scss"
     />
   </head>
+  <!-- #endregion -->
+  <body class="page__content">
+    <header class="page__header header">
+      <a
+        class="icon"
+        href="#logo"
+      >
+        <img
+          src="images/logo.png"
```

Comment:
Ensure that the image path 'images/logo.png' is correct and the image is available at this location. Otherwise, the logo might not display correctly.